src/content/docs/experiment/events.mdx

```diff
@@ -0,0 +1,230 @@
+
+Each time the agent receives a `201 outgoing payment created` response from the site visitor's wallet provider, the agent fires a `monetization` event on the browser window. 
+
+### Specifications
+
+<Specification anchor='events' />
+
+## Event listeners
+
+When a `monetization` event fires, there's no guarantee that payments will follow or, if they do, how often or how large the payments will be. 
+
+A `monetization` event indicates that the site visitor's wallet provider has created the resources needed for it to send a payment. 
+
+By adding an event listener to the relevant monetization `<link>` element (or one of its ancestors), you can use the `monetization` event properties to verify payments. These properties provide:
+
+* Information about the [amount and currency](#amountsent) of a sent (not received) payment
+* A URL representing an [incoming payment](#incomingpayment) that can be used to determine the amount received, similar to a receipt
+* The URL representing the [payment pointer or wallet address](#paymentpointer) that an incoming payment request was sent to
+
+### High-level flow
+
+<Steps>
+1. You add an event listener to a monetization `<link>`.
+2. The site visitor accesses your page.
+3. The site visitor's Web Monetization agent calls the Open Payments APIs to begin setting up the payment. More information about these API calls is provided in the [How payments work](/experiment/how-payments-work#a-deeper-dive-into-payments) page.
+4. The Web Monetization agent receives a `201 (outgoing payment created)` response from the site visitor's wallet provider.
+5. The Web Monetization agent fires a `monetization` event.
+6. Your event listener calls its defined function.
+</Steps>
+
+## Event properties
+
+### amountSent
+
+The `amountSent` property returns the `value` and `currency` of the sent payment. 
+
+* `value` - The amount. A <LinkOut href="https://www.w3.org/TR/payment-request/#dfn-valid-decimal-monetary-value">valid decimal monetary value</LinkOut> containing the amount that was sent.
+* `currency` - The currency code. A well-formed 3-letter <LinkOut href="https://www.iban.com/currency-codes">ISO4217</LinkOut> code that represents the currency that was sent, such as USD or GBP.
+
+```js title="Example"
+"value": "1.23"
+"currency": "USD"
+```
+
+:::note[Received amount]
+The `amountSent` is not necessarily the amount received. For example, if your payment account is set up to receive GBP, but your site visitor's account sends in USD, then exchange rates and currency conversion fees could affect the amount you receive.
+:::
+
+#### Example using `amountSent`
+
+```js
+<link rel="monetization" href="https://wallet.example.com/alice">
+<script>
+  const link = document.querySelector('link[rel="monetization"]')
+  link.addEventListener('monetization', (event) => {
```

Comment:
Would recommend adding an example with `window.addEventListener('monetization', ...)` and show that it covers all link tags on page `event.target` is the `HTMLLinkElement` for which event was fired.